include/config/general.h

```diff
@@ -73,6 +73,7 @@
 #define HQ_RANDOM                    TRUE    // If TRUE, replaces the default RNG with an implementation of SFC32 RNG. May break code that relies on RNG.
 #define COMPETITIVE_PARTY_SYNTAX     TRUE    // If TRUE, parties are defined in "competitive syntax".
 #define AUTO_SCROLL_TEXT             FALSE   // If TRUE, text will automatically scroll to the next line after NUM_FRAMES_AUTO_SCROLL_DELAY. Players can still press A_BUTTON or B_BUTTON to scroll on their own.
+#define CRY_AFTER_TITLE              TRUE    // If TRUE, Rayquaza's cry plays when A or Start is pressed at the title screen.
```

Comment:
I think this config should instead have you define what species the cry should play, instead of it just being rayquaza.
And if its SPECIES_NONE then its disabled.

Reply:
SPECIES is not defined when compiling this file. I don't feel like this belongs in pokemon.h but I may just have to move this there then.